<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TBTournamentDisplayWindow</class>
 <widget class="QMainWindow" name="TBTournamentDisplayWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tournament Display</string>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>500</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="mainLayout">
    <property name="spacing">
     <number>10</number>
    </property>
    <property name="leftMargin">
     <number>15</number>
    </property>
    <property name="topMargin">
     <number>15</number>
    </property>
    <property name="rightMargin">
     <number>15</number>
    </property>
    <property name="bottomMargin">
     <number>15</number>
    </property>
    <item>
     <!-- Tournament Header -->
     <widget class="QFrame" name="tournamentHeaderFrame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <layout class="QVBoxLayout" name="headerLayout">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="leftMargin">
        <number>15</number>
       </property>
       <property name="topMargin">
        <number>15</number>
       </property>
       <property name="rightMargin">
        <number>15</number>
       </property>
       <property name="bottomMargin">
        <number>15</number>
       </property>
       <item>
        <widget class="QLabel" name="tournamentNameLabel">
         <property name="text">
          <string>Tournament Name</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="styleSheet">
          <string>font-size: 18px; font-weight: bold;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="fundingSourcesLabel">
         <property name="text">
          <string>Funding Sources</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="styleSheet">
          <string>font-size: 14px;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <!-- Main Content Area: 3 columns (Left, Center, Right) with 1:2:1 ratio -->
     <layout class="QHBoxLayout" name="contentLayout">
      <property name="spacing">
       <number>15</number>
      </property>
      <item>
       <!-- Left Column: Statistics (top) and Chips (bottom) - Stretch factor 1 -->
       <widget class="QWidget" name="leftColumnWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QVBoxLayout" name="leftColumnLayout">
         <property name="spacing">
          <number>10</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
        <item>
         <!-- Upper Left: Tournament Statistics - Stretch to fill remaining space -->
         <widget class="QGroupBox" name="tournamentStatsGroupBox">
          <property name="title">
           <string>Tournament Statistics</string>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <layout class="QFormLayout" name="statsFormLayout">
           <property name="horizontalSpacing">
            <number>10</number>
           </property>
           <property name="verticalSpacing">
            <number>8</number>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="currentRoundLabel">
             <property name="text">
              <string>Current Round:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="currentRoundValue">
             <property name="text">
              <string>—</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="playersLeftLabel">
             <property name="text">
              <string>Players Left:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="playersLeftValue">
             <property name="text">
              <string>—</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="totalEntriesLabel">
             <property name="text">
              <string>Total Entries:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="totalEntriesValue">
             <property name="text">
              <string>—</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="averageStackLabel">
             <property name="text">
              <string>Average Stack:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLabel" name="averageStackValue">
             <property name="text">
              <string>—</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="elapsedTimeLabel">
             <property name="text">
              <string>Elapsed Time:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLabel" name="elapsedTimeValue">
             <property name="text">
              <string>—</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <!-- Lower Left: Chip Display - Size to fit content -->
         <widget class="QGroupBox" name="chipsGroupBox">
          <property name="title">
           <string>Tournament Chips</string>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QVBoxLayout" name="chipsLayout">
           <item>
            <widget class="QTableView" name="chipsTableView">
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectRows</enum>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::NoSelection</enum>
             </property>
             <property name="verticalScrollBarPolicy">
              <enum>Qt::ScrollBarAsNeeded</enum>
             </property>
             <property name="horizontalScrollBarPolicy">
              <enum>Qt::ScrollBarAsNeeded</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        </layout>
       </widget>
      </item>
      <item>
       <!-- Center Column: Controls (top) and Clock/Rounds (bottom) - Stretch factor 2 -->
       <widget class="QWidget" name="centerColumnWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>2</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QVBoxLayout" name="centerColumnLayout">
         <property name="spacing">
          <number>10</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
        <item>
         <!-- Upper Center: Tournament Controls - Fixed size to fit content -->
         <widget class="QGroupBox" name="controlsGroupBox">
          <property name="title">
           <string>Tournament Controls</string>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QVBoxLayout" name="controlsLayout">
           <property name="spacing">
            <number>10</number>
           </property>
           <item>
            <!-- Control Buttons -->
            <layout class="QHBoxLayout" name="clockButtonsLayout">
             <property name="spacing">
              <number>10</number>
             </property>
             <item>
              <widget class="QPushButton" name="previousRoundButton">
               <property name="text">
                <string>Previous Round</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pauseResumeButton">
               <property name="text">
                <string>Pause/Resume</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="nextRoundButton">
               <property name="text">
                <string>Next Round</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="callClockButton">
               <property name="text">
                <string>Call Clock</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <!-- Lower Center: Tournament Clock - Stretch to fill remaining space -->
         <widget class="QGroupBox" name="tournamentClockGroupBox">
          <property name="title">
           <string>Tournament Clock</string>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <layout class="QVBoxLayout" name="clockLayout">
           <property name="spacing">
            <number>10</number>
           </property>
           <item>
            <widget class="QLabel" name="tournamentClockLabel">
             <property name="text">
              <string>—</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="styleSheet">
              <string>font-size: 24px; font-weight: bold;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="currentRoundInfoLabel">
             <property name="text">
              <string>Current Round Information</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="nextRoundInfoLabel">
             <property name="text">
              <string>Next Round Information</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        </layout>
       </widget>
      </item>
      <item>
       <!-- Right Column: Results Display (full height) - Stretch factor 1 -->
       <widget class="QWidget" name="rightColumnWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QVBoxLayout" name="rightColumnLayout">
         <property name="spacing">
          <number>10</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
        <item>
         <!-- Results Display - Expand to fill all available space -->
         <widget class="QGroupBox" name="resultsGroupBox">
          <property name="title">
           <string>Tournament Results</string>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>400</height>
           </size>
          </property>
          <layout class="QVBoxLayout" name="resultsLayout">
           <item>
            <widget class="QTableView" name="resultsTableView">
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectRows</enum>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::SingleSelection</enum>
             </property>
             <property name="verticalScrollBarPolicy">
              <enum>Qt::ScrollBarAsNeeded</enum>
             </property>
             <property name="horizontalScrollBarPolicy">
              <enum>Qt::ScrollBarAsNeeded</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>